colossalai/benchmark/run.py

```diff
@@ -0,0 +1,86 @@
+import torch
+import inspect
+import os
+import subprocess
+import sys
```

Comment:
Should we put the benchmark files into the cli folder as well? I think this module is not gonna be used by the user directly, so better to hide it from the user.